CHANGELOG.md

```diff
@@ -283,6 +283,19 @@ Instead, contracts now revert with custom errors. Systems that interact with sma
 
 After 5.0, the storage location of some variables were changed. This is the case for `Initializable` and all the upgradeable contracts since they now use namespaced storaged locations. Any system relying on storage locations for retrieving data or detecting capabilities should be updated to support these new locations.
 
+## 4.9.6 (2024-02-29)
+
+- `Base64`: Fix issue where dirty memory located just after the input buffer is affecting the result. ([#4929](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4929))
+
+## 4.9.5 (2023-12-08)
+
+- `Multicall`: Make aware of non-canonical context (i.e. `msg.sender` is not `_msgSender()`), allowing compatibility with `ERC2771Context`. Patch duplicated `Address.functionDelegateCall` in v4.9.4 (removed).
```

Comment:
Given 4.9.4 doesn't exist, I'm mentioning both the multicall update and the duplicated line. I updated the [Github Release for 4.9.5](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v4.9.5) accordingly